### ISSUE TITLE
Loot roll bugs when someone disconnects in dungeon

### ISSUE THREAD

**Issue by oneadanut** [gamefreedomgit/Frostmourne#1886]
**Description:**
If someone disconnects during a loot roll, the item doesn't get distributed and people can't kick the guy because they get the message "can't kick during loot roll".

We opened a ticket and Aeradel couldn't kick the guy from party with any command, we had to finish the dungeon in 4 people.

**Comment by UltraNix** [comment]
@oneadanut Happens even when the roll timer is over?

**Comment by oneadanut** [comment]
yes

**Comment by Piootrek** [comment]
@oneadanut I wasn't able to reproduce this issue. Do you remember what item and dungeon it was?

**Comment by oneadanut** [comment]
@Piootrek I was in Azjol-Nerub, the guy went afk after joining the dungeon or maybe he got a crash but he was still online when we pulled first 2 packs of mobs before the watchers of first boss. The Stained-Glass Shard Ring dropped on trash, we all rolled for it with greed, the guy disconnected during the loot timer. The ring didn't get distributed to any player and we couldn't kick the guy from the dungeon because of the message "can't kick during loot roll" even the timer was long gone, because we even tried after we killed second boss.
